waspc/README.md

```diff
@@ -34,7 +34,32 @@ It can then also run that web app for you, deploy it (not yet but that is coming
 ### Setup
 We use the [`cabal`](https://cabal.readthedocs.io/) command line tool to build
 the project. The best way to install it is via
-[ghcup](https://www.haskell.org/ghcup/). The `ghcup tui` terminal user interface
+[ghcup](https://www.haskell.org/ghcup/) (if you're running an M1/M2 mac make sure to run it in rosetta).
+
+<details>
+<summary>M1/M2 rosetta setup</summary>
+
+1. Regular Shell: just run `arch -x86_64 zsh` in your terminal.
```

Comment:
This uses rosetta for the current terminal session?

Reply:
Yup, without any permanent changes